release/pkg/images/images.go

```diff
@@ -258,7 +258,11 @@ func GetReleaseImageURI(r *releasetypes.ReleaseConfig, name, repoName string, ta
 					semver = previousReleaseImageSemver
 					fmt.Printf("Image digest for %s image has not changed, tagging with previous dev release semver: %s\n", repoName, semver)
 				} else {
-					newSemver, err := filereader.GenerateNewDevReleaseVersion(previousReleaseImageSemver, "vDev", r.BuildRepoBranchName)
+					newBuildNumber, err := filereader.GetNewEksaDevBuildNumber(previousReleaseImageSemver, "vDev", r.BuildRepoBranchName)
+					if err != nil {
+						return "", errors.Cause(err)
+					}
+					newSemver, err := filereader.GetCurrentEksADevReleaseVersion(newBuildNumber, "vDev", r)
```

Comment:
I wonder if this could be passed in the `ReleaseConfig`? so we don't need to calculate it twice